storage/mysql/map_storage_test.go

```diff
@@ -190,6 +190,26 @@ var mapLeaf = trillian.MapLeaf{
 	ExtraData: []byte("Some Extra Data"),
 }
 
+func TestMapEmptyGet(t *testing.T) {
```

Comment:
Would you mind combining this and `TestMapGetUnknownKey` in single a table-driven test method? I've had a quick look and they seem close enough that it shouldn't cause you too much trouble.